jujugui/static/gui/src/app/models/handlers.js

```diff
@@ -430,11 +430,11 @@ YUI.add('juju-delta-handlers', function(Y) {
       if (hardwareCharacteristics) {
         data.hardware = {
           arch: hardwareCharacteristics.arch,
-          cpuCores: hardwareCharacteristics.cpucores,
-          cpuPower: hardwareCharacteristics.cpupower,
+          cpuCores: hardwareCharacteristics['cpu-cores'],
+          cpuPower: hardwareCharacteristics['cpu-power'],
           mem: hardwareCharacteristics.mem,
```

Comment:
The dot notation is preferred for JS objects, index notation being used when dot notation is not possible, i.e. when the property name has punctuation chars.